GAE/src/org/waterforpeople/mapping/app/web/rest/QuestionAnswerRestService.java

```diff
@@ -152,13 +154,24 @@ private void processApiV1Responses(List<QuestionAnswerStoreDto> responses,
         for (QuestionAnswerStoreDto qDto : responses) {
             String optionResponseValue = qDto.getValue();
```

Comment:
I had missed out this one.  `optionResponseValue` is confusing now since its processing image responses as well.